### DISCUSSION TITLE
How to use AutoOrtho for XP11 & XP12 on one PC[Question]

### DISCUSSION THREAD

**Discussion by janjacobs66** [kubilus1/autoortho#456] [General]
### Please check existing docs and issues first.

- [X] I have checked for existing answers
- [X] This is not a bug or other "Issue"
- [X] I have read the FAQ and documentation

### System Details

- OS: Windows 10
- XPlane Version: 11 & 12
- AutoOrtho Version [0.7.1]
- Filesystem type (NTFS)

### question

When I used the older AO version, I did the installation and pointed it to XP11.
I then created shortcuts in XP12, to XP11's Custom Scenery.
In XP & XP 12 AO worked / updated in flight etc

Now using v0.7.1, and creating shortcuts as before - no go
XP12 wont start, as its not happy with the scenery

Any workaround ?

**Comment by Jeff2122** [comment]
I have had a discussion on the .org with someone with the same exact problem. Unless you are the same person. 
I have to install XP11 again to prove it works. Yesterday I added AO to a previously untouched installation of XP12.07r1. It took about 1 minute and worked perfectly. I will get back to you if I have any hiccups.

**Comment by janjacobs66** [comment]
Hi Jeff

Yes, that’s me

**Comment by Jeff2122** [comment]
I did the XP11 experiment, and it went as expected. See my .org response. I included screenshots and my notes. See if there is anything that stands out as maybe a missed step. Other than that, maybe there is some security setting interfering somewhere.

**Reply by janjacobs66** [reply]
I did !
see my reply
Only difference is, I have XP11 as 1st, then added XP12

**Reply by Jeff2122** [reply]
Yeah. Sorry we couldn't get you going. But you have a backup that works for now. Take a break and maybe someday soon you'll try again and magically it will work.

**Reply by evilstiefel** [reply]
Could you at least provide a link to the solution if you already tested it and described it elsewhere? I'm sure there's more people that'd like to know, me included :).

**Reply by janjacobs66** [reply]
https://forums.x-plane.org/index.php?/forums/topic/296698-how-to-use-autoortho-for-xp11-xp12-on-one-pc/
